go.mod

```diff
@@ -76,5 +76,7 @@ require (
 	gopkg.in/tomb.v1 v1.0.0-20141024135613-dd632973f1e7 // indirect
 	gopkg.in/vmihailenco/msgpack.v2 v2.9.1 // indirect
 	gopkg.in/yaml.v2 v2.2.1 // indirect
+	labix.org/v2/mgo v0.0.0-20140701140051-000000000287 // indirect
```

Comment:
I think it is because of dependencies:

```
% GO111MODULE=on go mod why launchpad.net/gocheck
# launchpad.net/gocheck
github.com/zalando/skipper/swarm
github.com/hashicorp/memberlist
github.com/hashicorp/go-msgpack/codec
github.com/hashicorp/go-msgpack/codec.test
labix.org/v2/mgo/bson
labix.org/v2/mgo/bson.test
launchpad.net/gocheck

% GO111MODULE=on go mod why labix.org/v2/mgo/bson
# labix.org/v2/mgo/bson
github.com/zalando/skipper/swarm
github.com/hashicorp/memberlist
github.com/hashicorp/go-msgpack/codec
github.com/hashicorp/go-msgpack/codec.test
labix.org/v2/mgo/bson
```

Reply:
I will test to delete labix.org/v2/mgo dependency, maybe it helps CDP

Reply:
:+1: